wikitemplate.md

```diff
@@ -15,6 +15,7 @@
   - [ ] opened tabs can be reloaded
   - [ ] bookmarks on the bookmark toolbar can be opened
   - [ ] bookmarks in the bookmark folder toolbar can be opened
+  - [ ] ensure upgrade sets fingerprint to `Block all fingerprint` when global fingerprint is enabled
```

Comment:
nitpick: should be using the same one as `wikitemplate-cr-upgrade.md`, example:

"ensure upgrading from `0.19.x` to `0.20.x` sets fingerprint to `Block all fingerprint` when global fingerprint is enabled"